systemtests/src/test/java/org/bf2/admin/kafka/systemtest/plain/RecordEndpointTestIT.java

```diff
@@ -229,23 +230,24 @@ void testConsumeRecordsByStartingOffset(int startingOffset, int expectedResults)
 
     @ParameterizedTest
     @CsvSource({
-        "1",
-        "2",
-        "3",
-        "4",
-        "20"
+        "10",
+        "20",
+        "30",
+        "40",
+        "200"
     })
     void testConsumeLatestRecords(int limit) {
         final String topicName = UUID.randomUUID().toString();
-        final int totalRecords = 10;
-        topicUtils.createTopics(List.of(topicName), 1, Status.CREATED); // single partition
+        final int totalRecords = 20;
```

Comment:
Try using `100` here. That way the cases for `limit`s less than `200` will be selecting fewer than the total like the previous version.
```suggestion
        final int totalRecords = 100;
```